libs/domain/push-notification/src/services/push-notification.providers.ts

```diff
@@ -0,0 +1,12 @@
+import { Provider } from '@spryker-oryx/di';
+import { DefaultPushService } from './default-push.service';
+import { webPushProviders } from './providers';
+import { PushService } from './push.service';
+
+export const pushNotificationProviders: Provider[] = [
+  {
+    provide: PushService,
+    useClass: DefaultPushService,
+  },
+  ...webPushProviders,
```

Comment:
PushNotifications feature should only provide PushService and no other push providers.
And another more specific features should extend PushNotificationFeature and provide their specific push providers.